manifests/sysmgmt.yaml

```diff
@@ -83,15 +83,15 @@ spec:
     namespace: services
   - name: cray-hms-scsd
     source: csm-algol60
-    version: 3.0.0
+    version: 3.0.1
     namespace: services
     swagger:
     - name: scsd
       version: v1
       url: https://raw.githubusercontent.com/Cray-HPE/hms-scsd/v1.19.0/api/openapi.yaml
   - name: cray-hms-rts
     source: csm-algol60
-    version: 4.0.2
+    version: 5.0.0
```

Comment:
@jwlv Should line #99 below also be changed to `5.0.0`?

Reply:
Good catch.  Will create new PR.